Makefile

```diff
@@ -17,8 +17,12 @@ deploy:
 import: package
 	-$(GRAVITY) app delete --ops-url=$(OPS_URL) $(REPOSITORY)/$(NAME):$(VERSION) \
 		--force --insecure
-	$(GRAVITY) app import --debug --vendor --glob=**/*.yaml --registry-url=apiserver:5000 \
-		--set-image=log-collector:$(VERSION) --set-image=log-forwarder:$(VERSION) \
-		--ops-url=$(OPS_URL) --repository=$(REPOSITORY) --name=$(NAME) \
-		--version=$(VERSION) --insecure .
+	$(GRAVITY) app import --debug --insecure --vendor --glob=**/*.yaml \
+		--registry-url=apiserver:5000 --ops-url=$(OPS_URL) \
+		--set-image=log-collector:$(VERSION) --set-image=log-forwarder:$(VERSION) --set-image=log-tailer:$(VERSION) \
```

Comment:
Minor nitpick: Maybe collect some or all of these args in a variable (especially `--set-image` ones).